spec/app_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../app' 
+  
+def reset_table
+  seed_sql = File.read('spec/table_seed.sql')
+  connection = PG.connect({ host: '127.0.0.1', dbname: 'shop_manager' })
+  connection.exec(seed_sql)
+end
+
+describe Application do
+  before(:each) do
+    reset_table
+  end
+  it "should show the user options" do
+    io = double :io
+    app = Application.new('shop_manager', io, ItemRepository.new, OrderRepository.new)
```

Comment:
`ItemRepository.new` and `OrderRepository.new` can be doubles too in a unit test.